src/chapters/07/main.tex

```diff
@@ -462,32 +461,33 @@ \section{Solving with R}\label{sec:solving-with-R}
 \mintinline{R}{run}, \mintinline{R}{take_turn} and
 \mintinline{R}{mean_satisfaction}.
 
-The \mintinline{R}{initialize} method is run at the time the object is first
+The \mintinline{R}{initialise} method is run at the time the object is first
```

Comment:
`initialize`

it's a built in R6 function, so spelt the American way

Reply:
I am amazed that that's something I changed:

A. I never notice stuff like that.
B. Changing the name of a library is obviously dumb.

What an idiot. 🤦